src/Microsoft.Extensions.DependencyModel/RuntimeLibrary.cs

```diff
@@ -8,12 +8,23 @@ namespace Microsoft.Extensions.DependencyModel
 {
     public class RuntimeLibrary : Library
     {
-        public RuntimeLibrary(string libraryType, string packageName, string version, string hash, string[] assemblies, Dependency[] dependencies, bool serviceable)
+        public RuntimeLibrary(
+            string libraryType,
+            string packageName,
+            string version,
+            string hash,
+            RuntimeAssembly[] assemblies,
+            RuntimeTarget[] subTargets,
+            Dependency[] dependencies,
+            bool serviceable)
             : base(libraryType, packageName, version, hash, dependencies, serviceable)
         {
-            Assemblies = assemblies.Select(path => new RuntimeAssembly(path)).ToArray();
+            Assemblies = assemblies;
+            SubTargets = subTargets;
```

Comment:
We call it `RuntimeTargets` elsewhere (to match the new name in lock.json), I assume this just needs to be updated to match.